povertymapping/iso3.py

```diff
@@ -1,30 +1,44 @@
 
 from functools import lru_cache
 import pandas as pd
+import warnings
 # ISO3166 source list
 # see https://github.com/lukes//ISO-3166-Countries-with-Regional-Codes
 
-ISO3_URL = 'https://raw.githubusercontent.com/lukes/ISO-3166-Countries-with-Regional-Codes/master/all/all.csv'
-
+ISO3_URL = "https://datahub.io/core/country-codes/r/country-codes.csv"
 
 @lru_cache(maxsize=None)
 def get_iso3_codes():
     iso3_df = pd.read_csv(ISO3_URL)
-    iso3_lookup = {item['name'].lower(): item for item in iso3_df.to_dict('records')}
+    iso3_df.columns = [col.lower().replace('_','-').replace(' ','-') for col in iso3_df.columns]
+    iso3_lookup = iso3_df[['iso3166-1-alpha-3','cldr-display-name']].apply(lambda x: x.astype(str).str.lower())
+    iso3_lookup['cldr-display-name'] = iso3_lookup['cldr-display-name'].str.replace(' ','-')
+    iso3_lookup['iso-alpha-2'] = iso3_lookup['iso3166-1-alpha-3'].apply(lambda x: x[:-1])
+    iso3_lookup.columns = ['iso-alpha-3','common-name','iso-alpha-2']
```

Comment:
where does common name come from? can't seem to find it in the CSV file?

Reply:
original column name is `cldr-display-name`, just renamed it for readability